school_evaluation/models/school_evaluation.py

```diff
@@ -164,20 +163,33 @@ class SchoolEvaluationTemplate(models.Model):
     type = fields.Selection([('faculty', 'Faculty'), ('student', 'Student')],
                             'User Type', required=True, default='faculty',
                             help="Type of Evaluation")
-    rating_line = fields.One2many('rating.rating', 'rating_id', 'Rating')
+    rating_line_ids = fields.Many2many('rating.rating', 'rating_table')
 
 
 class RatingRating(models.Model):
-    _name = 'rating.rating'
-    _description = "Rating details"
-    _rec_name = 'point'
-    _order = "point desc"
-
-    rating_id = fields.Many2one('school.evaluation.template', 'Stud',
-                                help="Ratings")
-    point = fields.Integer('Rating in points', required=True,
-                           help="Points")
-    rating = fields.Char('Remarks', required=True)
+    _inherit = 'rating.rating'
+
+    @api.model
+    def create(self, vals):
+        """Set Document model name for rating."""
+        res_model = self.env['ir.model'].search([
+            ('model', '=', 'school.evaluation.template')])
+        vals.update({'res_model_id': res_model.id})
+        res = super(RatingRating, self).create(vals)
+        return res
+
+    @api.one
```

Comment:
sure api.one?

Reply:
Yes. This method is overridden from the base and there's api.one.